src/frontend/typer/totrinnskontroll.ts

```diff
@@ -0,0 +1,26 @@
+export enum TotrinnskontrollStatus {
```

Comment:
Ønsker feedback på disse ulike navnene, og feltnavn i TotrinnskontrollResponse. 

```
case TotrinnskontrollStatus.IKKE_AUTORISERT:
            return <SendtTilBeslutter totrinnskontroll={totrinnskontroll.data.totrinnskontroll} />;
```

Reply:
IKKE_AUTORISERT så tenker jeg at innlogget bruker ikke kan utføre totrinnskontroll her... Høres jo greit ut for min del.

Reply:
Jeg tenkte mest på navnen i de ulike typene/interfacen :P  `totrinnskontroll.data.totrinnskontroll`

Kodedelen var ett eksempel på hvordan det ser ut: Ja det er når en saksbehandler ikke er BEHANDLER eller den samme som sendte behandlingen til besluttning :)

Reply:
Aha, `totrinnskontroll.data.status` , kanskje?

Reply:
data inneholder status og totrinnskontroll nå :P